tests/workflow-scripts.test.sh

```
@@ -252,6 +252,175 @@ else
+fi
+
+# Inverse fixtures for #396.4 — this ratchet went vacuous once by being
+# line-scoped; these rows make it impossible for it to go vacuous the same way
+# twice. The fixture reproduces the SHAPE of the real blind spot: the call split
+# across two lines, with the nested `path.basename(file)` that defeats a
+# `[^)]`-bounded scan. Built in $TMPDIR_FIXTURES — the real harness on disk is
+# never mutated.
+BUDGET_FIXTURE_BAD="$TMPDIR_FIXTURES/budget-literal-multiline.js"
+{
+  echo '      const { errors, record } = await runScript('
+  echo '        source, { args: argsValue }, `${path.basename(file)} [args:${shapeName}]`, 2000);'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)